packages/blocks/src/api/validation.js

```diff
@@ -390,7 +409,12 @@ export function getNextNonWhitespaceToken( tokens ) {
  */
 export function isEquivalentHTML( actual, expected ) {
 	// Tokenize input content and reserialized save content
-	const [ actualTokens, expectedTokens ] = [ actual, expected ].map( tokenize );
+	const [ actualTokens, expectedTokens ] = [ actual, expected ].map( getHTMLTokens );
```

Comment:
I'm inclined to think this change could be simplified to something like:

```js
try {
	const [
		actualTokens,
		expectedTokens,
	] = [ actual, expected ].map( tokenize );
} catch ( error ) {
	return false;
}
```

Logging the warning about the specific string becomes a bit trickier. We could still have `getHTMLTokens` which catches the specific failure, logs, then `throw`s up to the `catch` here.

Or we could keep as-is. My only thought was avoiding the overloaded return value, where `false` is a bit of a semantically ambiguous value.

Reply:
AFAIK the const's are scoped to the try block so I don't think is possible unless they were converted to `let` and defined outside, and it seemed better to keep the constness.

I've cleaned it up a bit already, and happy to make a further change, but otherwise will go as-is.